apps/api/src/app/shared/cache/cached-interceptor.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect } from 'chai';
+import { ICacheService, Cached, InvalidateCache } from '@novu/dal';
+import { CacheService } from './cache-service.spec';
+import { beforeEach } from 'mocha';
+
+describe('cached interceptor', function () {
```

Comment:
here you need to mock the `CacheService` using the `jest`: https://jestjs.io/docs/es6-class-mocks#manual-mock
before each test you have to clear the mock: https://jestjs.io/docs/jest-object#jestclearallmocks to make sure that all the mock counts were reset

then the tests should verify whether the `Cached` logic was called and the mocked functions on the mocked `CacheService` were called or not...